.devcontainer/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM centos/devtoolset-6-toolchain-centos7:latest
+
+LABEL \
+  description="Ovsrpro Centos 7 Development Container" \
+  name="ovsrpro-dev"
+
+USER 0
+RUN set -eux; \
+  yum --assumeyes update; \
+  yum install --assumeyes \
+    epel-release \
+    # This repository gives us more modern versions of some software, such as Git.
+    https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.7-1.x86_64.rpm; \
+  yum install --assumeyes \
+    cmake3 \
+    git \
+    make \
+    wget; \
+  # yum --assumeyes clean all; \
+  ln -s cmake3 /usr/bin/cmake; \
+  ln -s ctest3 /usr/bin/ctest; \
+  ln -s cpack3 /usr/bin/cpack; \
+  yum install --assumeyes \
```

Comment:
Why do 2 separate install steps? I think you should combine this step with line 14.

Reply:
I prefer to have related functionality close together.  Hence creating the symlinks soon after installing CMake. Also, the first yum install is just necessary tools; the second is dependencies for specific ovsrpro projects. I'll change it, though.